build/dependencies.props

```diff
@@ -6,6 +6,7 @@
     <!-- Azure SignalR -->
     <MessagePackPackageVersion>1.7.3.4</MessagePackPackageVersion>
     <MicrosoftAspNetCoreHttpConnectionsClientPackageVersion>1.0.0</MicrosoftAspNetCoreHttpConnectionsClientPackageVersion>
+    <MicrosoftAspNetCoreHttpPackageVersion>2.1.0</MicrosoftAspNetCoreHttpPackageVersion>
```

Comment:
I think we should consider dropping this dependency and just use a dictionary instead.

Reply:
Yeah, we can do this.